spec/features/publish/courses/new_tda_course_spec.rb

```diff
@@ -250,20 +273,44 @@ def then_i_am_on_the_choose_schools_page
     expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_schools_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
   end
 
+  def and_the_back_link_points_to_outcome_page
+    and_i_click_back
+    expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_outcome_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
+    and_i_click_continue
+  end
+
   def when_i_choose_the_school
     check provider.sites.first.location_name
     and_i_click_continue
   end
 
+  def then_the_back_link_points_to_the_school_page
+    and_i_click_back
+    expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_schools_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
+    and_i_click_continue
+  end
+
   def and_i_choose_the_study_site
     check provider.study_sites.first.location_name
     and_i_click_continue
   end
 
+  def and_the_back_link_points_to_the_study_site_page
+    and_i_click_back
+    expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_study_sites_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
+    and_i_click_continue
+  end
+
   def then_i_am_on_the_add_applications_open_date_page
     expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_applications_open_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
   end
 
+  def and_the_back_link_points_to_applications_open_date_page
+    and_i_click_back
```

Comment:
Is it possible to verify the url of the back links instead of clicking back and checking the current path? They feel like unnecessary extra steps.

Reply:
Done